api/apiclient.go

```diff
@@ -658,9 +711,37 @@ func dialWebsocketMulti(ctx context.Context, addrs []string, path string, opts d
 	return result.(*dialResult), nil
 }
 
+func lookupIPAddr(ctx context.Context, host string, resolver IPAddrResolver) ([]string, error) {
+	addrs, err := resolver.LookupIPAddr(ctx, host)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	ips := make([]string, 0, len(addrs))
+	for _, addr := range addrs {
+		if addr.Zone != "" {
+			// Ignore IPv6 zone. Hopefully this shouldn't
+			// cause any problems in practice.
+			logger.Infof("ignoring IP address with zone %q", addr)
+			continue
+		}
+		ips = append(ips, addr.IP.String())
```

Comment:
Wouldn't it be saner to put those addresses on the list in '%' notation?
ip := addr.IP.String()
if addr.Zone != "" { 
  ip = fmt.Sprintf("%s%%%s", o[, addr.Zone) 
} 
ips.append(ips, ip)

(although I think that it wouldn't be a problem too, since I can't think of a way we'd get a zoned address from resolver)